### ISSUE TITLE
Factor 2 in SMEFT running of Cee?

### ISSUE THREAD

**Issue by lukallwi** [wilson-eft/wilson#111]
Switching on Ced_1211 = 1 at 1 TeV and running to mZ, wilson finds Cee_1211 (mZ) = -0.00254, while doing the same with DsixTools one finds -0.0013, i.e. one half of that, which seems also to be consistent with the anomalous dimensions in 1312.2014.
Could this be an issue with symmetry factors due to Cee_1211 = Cee_1112?

**Comment by peterstangl** [comment]
Hi Lukas, what exactly do you mean by "Cee_1211"? This coefficient is not part of the non-redundant WCxf basis (see [here](https://wcxf.github.io/assets/pdf/SMEFT.Warsaw.pdf) for a list of coefficients), so wilson shouldn't return this coefficient at all. Do you go to the redundant basis and symmetrise to obtain Cee_1211?

**Comment by peterstangl** [comment]
If you want to go from the non-redundant WCxf basis to the redundant symmetrized basis, this can be done is wilson as follows:
```python
from wilson import Wilson
from wilson.util import smeftutil

Wilson.set_default_option('smeft_accuracy', 'leadinglog')

w = Wilson({'ed_1211':1}, scale=1e3, eft='SMEFT', basis='Warsaw')
wc_run = w.match_run(91.1876, 'SMEFT', 'Warsaw')
Cee_symmetrized = smeftutil.wcxf2arrays_symmetrized(wc_run.dict)['ee']

Cee_symmetrized[0,1,0,0] # should give -0.0013
```

An unrelated issue is that in this example 'smeft_accuracy' set to 'leadinglog' gives a negative `Cee`, while 'smeft_accuracy' set to 'integrate' gives a positive one. But I think this problem is related to the fact that Ced_1211 = 1 is huge (it is 10^6/TeV^2)! This problem disappears if a more reasonable value like Ced_1211 = 1e-6 is used.

**Comment by lukallwi** [comment]
Right, my bad, I had them swapped, indeed only Cee_1112 exists. So it's Ced_1211 running into Cee_1112. Looking at appendix C.8.2 in https://arxiv.org/pdf/1312.2014.pdf it would seem that there are two contributions from Ced, one with \delta_pr, and one with \delta_st, to the running of Cee. So if prst = 1112 I would expect only one of the two to contribute, whiletThe result from wilson seems to be consistent with taking both terms. But maybe this could be indeed an issue related to the fact that the coefficient is big

**Comment by peterstangl** [comment]
I think this is unrelated to the size of the coefficient. Note that appendix C.8.2 in https://arxiv.org/pdf/1312.2014.pdf uses the redundant symmetrized basis. And in this basis you get two contributions of the same size to CRee_1211 and CRee_1112 (denoting the coefficients in the redundant basis as CR). If you then go to the non-redundant WCxf basis, you get Cee_1112=CRee_1211+CRee_1112 while Cee_1211 is not part of the basis. Isn't everything consistent then?
